src/backend/dx11/src/conv.rs

```diff
@@ -102,3 +112,210 @@ pub fn map_rect(rect: &Rect) -> D3D11_RECT {
         bottom: rect.h as _,
     }
 }
+
+pub fn map_topology(primitive: Primitive) -> D3D11_PRIMITIVE_TOPOLOGY {
```

Comment:
another thing to share with dx12? assuming `D3D11_PRIMITIVE_TOPOLOGY == D3D_PRIMITIVE_TOPOLOGY`

Reply:
Yup